### PR TITLE
Make import of data from classroom.csv and classroom2.csv consistent in tidy-data.Rmd vignette

### PR DESCRIPTION
**Pull request by ramiromagno**
In the vignette tidy-data.Rmd, when you read the example data from classroom2.csv:

```R
read.csv("classroom2.csv", stringsAsFactors = FALSE)
```

Suzy's grade on quiz1 should be `"F"` but it's read as `FALSE`. That's because of `read.csv()` defaults. This fix is needed for consistency with the previous example:

```R
classroom <- read.csv("classroom.csv", stringsAsFactors = FALSE)
```

### CODE REVIEW

**Comment by hadley** [comment]
I think it would be better to just replace with inline calls to `tribble()`

**Comment by ramiromagno** [comment]
hi Hadley,

Thanks for leaving the opportunity to contribute.

File: vignettes/tidy-data.Rmd [hunk 1]
```diff
@@ -37,14 +37,26 @@ Like families, tidy datasets are all alike but every messy dataset is messy in i
 Most statistical datasets are data frames made up of **rows** and **columns**. The columns are almost always labeled and the rows are sometimes labeled. The following code provides some data about an imaginary classroom in a format commonly seen in the wild. The table has three columns and four rows, and both rows and columns are labeled.
 
 ```{r}
-classroom <- read.csv("classroom.csv", stringsAsFactors = FALSE)
+library(tibble)
+classroom <- tribble(
```
Comment (hadley) [inline vignettes/tidy-data.Rmd:41]: Do you mind formatting these so that the columns line up?

Comment (ramiromagno) [reply vignettes/tidy-data.Rmd:41]: of course.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**